common/src/test/java/com/vexsoftware/votifier/util/ArgsToVoteTest.java

```diff
@@ -37,72 +27,28 @@ void handlesSingleUsernameTooLong() {
         });
     }
 
-    @ParameterizedTest
```

Comment:
Why rewrite these as individual tests? That's not in the scope of this PR.

Reply:
#295